src/style.css

```diff
@@ -0,0 +1,148 @@
+/* #region stars */
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 96;
+  height: 96px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(n) {
+  background-image: url(images/star-active.svg);
+}
+
+/* #endregion */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.image {
+  padding: 0;
+  margin: 32px 19px 36px;
+}
+
+.image__item {
+  padding: 0;
+  height: 134px;
+  width: 160px;
+}
+
+.computer-name {
+  font-style: normal;
+  color: #060b35;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 16px 4px;
```

Comment:
Almost all of your elements have the same 16px horizontal margin to card border. I think it will be better solution, if you add padding: 16px to your product_card block and then add extra horizontal margins to elements only if it is neccesary.